packages/kolibri-common/components/syncComponentSet/SelectDeviceModalGroup/SelectDeviceForm.vue

```diff
@@ -120,7 +120,7 @@
       </KFixedGrid>
     </template>
 
-    <KButtonGroup class="under-buttons">
+    <div class="under-buttons">
```

Comment:
KButtonGroup was introducing a weird `margin-bottom: -8px` that was making the scrollbar appear.